src/com/fundation/search/common/SearchLogger.java

```diff
@@ -56,27 +42,12 @@ private SearchLogger() {
         // Logger
         logger = Logger.getLogger(SearchLogger.class.getName());
 
-        // Currently date in "YYYY-MMM-dd" format.
-        currently_date = new SimpleDateFormat("YYYY-MMM-dd").format(new Date());
-
-        // This block configure the logger with handler and formatter
         try {
-            fileHandler = new FileHandler(System.getProperty("user.dir")
-                    + SYSTEM_SEPARATOR //System separator
-                    + "logs" // folder to save .log
-                    + SYSTEM_SEPARATOR //System separator
-                    + currently_date // currently date on String format
-                    + ".log", true);
-
+            LogManager.getLogManager().readConfiguration(
+                    new FileInputStream(".\\config\\logging.properties"));
```

Comment:
Please use AbsolutePath and system separator:
private static final String SEPARATOR = System.getProperty("file.separator");